pkg/registry/generic/etcd/etcd.go

```diff
@@ -271,10 +272,21 @@ func (e *Etcd) Update(ctx api.Context, obj runtime.Object) (runtime.Object, bool
 	if err != nil {
 		return nil, false, err
 	}
+	// If the object specified by the user does not have a resource version,
+	// then populete it with the latest version.
+	// Else, we check that the version specified by the user matches the version of latest etcd object.
+	useLatestResourceVersion := false
```

Comment:
I would call this unsafeUpdate, unconditionalUpdate, forceUpdate, or something like that.